### PR TITLE
added three new rpc tests - two for Sapling and one for getNormalizedScript 

### LINKED ISSUE #1356: Update the rpc nodes test
Add test cases for the `RpcClient` methods that are not included in the `rpc-nodes.spec.ts` file: https://github.com/ecadlabs/taquito/blob/master/integration-tests/rpc-nodes.spec.ts

Tests needed for the following methods:
- getSaplingDiffById
- getSaplingDiffByContract
- getNormalizedScript

Validate if other methods are missing from the test and add tests case accordingly.

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Closes #1356 1356

### CODE REVIEW

File: integration-tests/contract-originate-contract-with-sapling.spec.ts [hunk 1]
```diff
@@ -34,6 +34,8 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
         ledger2: SaplingStateAbstraction
       }
 
+      console.log("Sapling Contract address "+contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/contract-originate-contract-with-sapling.spec.ts:37]: console.log can be removed

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/config.ts [hunk 2]
```diff
@@ -71,11 +73,12 @@ const ithacanetEphemeral = {
 };
 
 const hangzhounetEphemeral = {
-  rpc: process.env['TEZOS_RPC_HANGZHOUNET'] || 'https://hangzhounet.api.tez.ie',
+  rpc: process.env['TEZOS_RPC_HANGZHOUNET'] || 'http://10.60.41.80:8732/',
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/config.ts:76]: restored

File: integration-tests/contract-originate-contract-with-sapling.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/contract-originate-contract-with-sapling.spec.ts:37]: ooops thanks :) removed.

File: integration-tests/config.ts [hunk 3]
```diff
@@ -85,11 +88,12 @@ const hangzhounetEphemeral = {
 };
 
 const ithacanetFaucet = {
-  rpc: process.env['TEZOS_RPC_ITHACANET'] || 'https://ithacanet.ecadinfra.com/',
+  rpc: process.env['TEZOS_RPC_ITHACANET'] || 'http://ecad-ithacanet-full.i.tez.ie:8732/',
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/config.ts]: fixed

File: integration-tests/rpc-nodes.spec.ts [hunk 4]
```diff
@@ -5,309 +5,343 @@ import { encodeExpr } from '@taquito/utils';
...
+          done();
+        });
+
+        it(`Access the code and data of a contract`, async (done) => {
+          const script = await rpcClient.getScript(knownContract);
+          expect(script).toBeDefined();
+          done();
+        });
+
+        it(`Access the script of the contract and normalize it using the requested unparsing mode`, async (done) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/rpc-nodes.spec.ts:66]: removed

**Pull request merged**